test/PrizePool.t.sol

```diff
@@ -39,7 +39,7 @@ contract PrizePoolTest is Test {
         vm.warp(1000 days);
```

Comment:
The TWAB Lib has a public `PERIOD_OFFSET` that is used to compute the storage periods and align them with draw frequencies. There's a hard requirement that this `PERIOD_OFFSET` is in the past. Given that, most tests that use the TWAB Controller should have a `vm.warp(TwabLib.PERIOD_OFFSET);` at the start. 
We also need draws to line up with the periods, it looks like `drawPeriodSeconds = 1 days;` (periods are currently hardcoded to `1 days` so there will be one observation record per draw with this setup) so that is safe atm but quite flaky.

You mentioned a while ago to keep these values hardcoded. There's some minor gas improvements but definitely a hit to the dev experience. Should we make this a configurable parameter in the TWAB Controller constructor & just funnel it through to the TWAB Lib?

Reply:
Yes, I think the constructor is a great option.  Especially if the field is immutable, because it will be as gas-efficient as a constant but can also be public, so that other contracts can query it